### PR TITLE
Add nodejs-optimize-css-assets-webpack-plugin package

### PR DESCRIPTION
**Pull request by amirfefer**

### CODE REVIEW

File: comps/comps-foreman-rhel7.xml [hunk 1]
```diff
@@ -34,6 +34,7 @@
       <packagereq type="default">foreman-service</packagereq>
       <packagereq type="default">foreman-telemetry</packagereq>
       <packagereq type="default">foreman-vmware</packagereq>
+      <packagereq type="default">nodejs-optimize-css-assets-webpack-plugin</packagereq>
```
Comment (evgeni) [inline comps/comps-foreman-rhel7.xml:37]:
```suggestion
      <packagereq type="default">tfm-nodejs-optimize-css-assets-webpack-plugin</packagereq>
```

and then this needs proper sorting :)

**Comment by evgeni** [comment]
could you also please omit the ordering changes around google-cloud-env?

File: comps/comps-foreman-rhel7.xml (hunk 1 above)
Comment (evgeni) [reply comps/comps-foreman-rhel7.xml:37]: and `nodejs-optimize-css-assets-webpack-plugin` (without the `tfm-` prefix) needs adding to `comps-foreman-el8.xml` (the script currently fails to do that for two comps files :()

File: rel-eng/tito.props [hunk 2]
```diff
@@ -89,6 +89,7 @@ whitelist = foreman
   nodejs-node-gyp
   nodejs-node-sass
   nodejs-object-assign
+  nodejs-optimize-css-assets-webpack-plugin
```
Comment (evgeni) [inline rel-eng/tito.props:92]: same needs to be added to `foreman-nightly-el8`, our scripts fail to do so :(

**Review by evgeni — CHANGES_REQUESTED** [review]
comments inline

**Comment by amirfefer** [comment]
Thanks @evgeni !
Updated

**Comment by amirfefer** [comment]
Rebased

**Pull request merged**